lib/ansible/modules/crypto/luks_device.py

```diff
@@ -78,12 +81,20 @@
             - "Removes given key from the container on I(device). Does not
               remove the keyfile from filesystem.
               Parameter value is the path to the keyfile with the passphrase."
-            - "BEWARE that it is possible to remove even the last key from the
-              container. Data in there will be irreversibly lost
-              without a warning."
+            - "NOTE that removing keys is I(not idempotent). Trying to remove
+              a key which no longer exists results in an error."
+            - "NOTE that to remove the last key from a LUKS container, the
+              I(allow_to_remove_last_key) option must be set to C(yes)."
             - "BEWARE that working with keyfiles in plaintext is dangerous.
               Make sure that they are protected."
         type: path
+    allow_to_remove_last_key:
```

Comment:
```suggestion
    force_remove_last_key:
```

Reply:
Good suggestion! I changed the PR accordingly. @Xyon is the new name OK for you as well?